experiments/golden-results/StratoX-summary.txt

```diff
@@ -1,33 +1,28 @@
-Occurs: 204 times
-Calling function: Process_Pragma_Declaration
-Error message: Unsupported pragma: Postcondition
```

Comment:
Why have the number of postconditions hit fallen?  I can't see that subtyping constraints should affect this?

Reply:
Is it that these are being converted to crashes?  If so ... is that intentional?

Reply:
Would be good to know, yes - but I'm not seeing evidence that there is suddenly more crashes either.

Reply:
Ah, so it actually looks like these disappeared in #280